.gitignore

```diff
@@ -4,7 +4,7 @@
 
 # misc
 .DS_Store
-/dist/
+/dist/*.map
```

Comment:
Why do we need to publish the library build to git? I'd prefer to keep `/dist/` here and only have the build inside the npm package.

Reply:
removed, published by accident, was a hot fix of sorts.